IBPSA/Fluid/Actuators/BaseClasses/PartialDamperExponential.mo

```diff
@@ -1,74 +1,116 @@
+    (2 / rho_default / k1 * dpDamper_nominal)^0.5
+    "Nominal face velocity";
+  final parameter Modelica.SIunits.Area A=m_flow_nominal/rho_default/v_nominal
     "Face area";
-
- parameter Boolean roundDuct = false
+  parameter Boolean roundDuct = false
     "Set to true for round duct, false for square cross section"
-   annotation(Dialog(enable=not use_deltaM));
- parameter Real ReC=4000 "Reynolds number where transition to turbulent starts"
-   annotation(Dialog(enable=not use_deltaM));
- parameter Real a(unit="")=-1.51 "Coefficient a for damper characteristics"
-  annotation(Dialog(tab="Damper coefficients"));
- parameter Real b(unit="")=0.105*90 "Coefficient b for damper characteristics"
-  annotation(Dialog(tab="Damper coefficients"));
- parameter Real yL = 15/90 "Lower value for damper curve"
-  annotation(Dialog(tab="Damper coefficients"));
- parameter Real yU = 55/90 "Upper value for damper curve"
-  annotation(Dialog(tab="Damper coefficients"));
- parameter Real k0(min=0) = 1E6
-    "Flow coefficient for y=0, k0 = pressure drop divided by dynamic pressure"
-  annotation(Dialog(tab="Damper coefficients"));
- parameter Real k1(min=0) = 0.45
-    "Flow coefficient for y=1, k1 = pressure drop divided by dynamic pressure"
-  annotation(Dialog(tab="Damper coefficients"));
- parameter Boolean use_constant_density=true
+    annotation(Dialog(enable=not use_deltaM));
+  parameter Real ReC=4000 "Reynolds number where transition to turbulent starts"
+    annotation(Dialog(enable=not use_deltaM));
+  parameter Real a(unit="1")=-1.51 "Coefficient a for damper characteristics"
+    annotation(Dialog(tab="Damper coefficients"));
+  parameter Real b(unit="1")=0.105*90 "Coefficient b for damper characteristics"
+    annotation(Dialog(tab="Damper coefficients"));
+  parameter Real yL(unit="1") = 15/90 "Lower value for damper curve"
+    annotation(Dialog(tab="Damper coefficients"));
+  parameter Real yU(unit="1") = 55/90 "Upper value for damper curve"
+    annotation(Dialog(tab="Damper coefficients"));
+  final parameter Real k0(min=0, unit="1") = 2 * rho_default * (A / (l * kDamMax))^2
+    "Loss coefficient for y=0, k0 = pressure drop divided by dynamic pressure"
+    annotation(Dialog(tab="Damper coefficients"));
+  parameter Real k1(min=0, unit="1") = 0.45
+    "Loss coefficient for y=1, k1 = pressure drop divided by dynamic pressure"
+    annotation(Dialog(tab="Damper coefficients"));
+  parameter Real l(min=1e-10, max=1) = 0.0001
+    "Damper leakage, ratio of flow coefficients k(y=0)/k(y=1)"
+    annotation(Dialog(tab="Damper coefficients"));
+  parameter Boolean use_constant_density=true
     "Set to true to use constant density for flow friction"
-   annotation (Evaluate=true, Dialog(tab="Advanced"));
- Medium.Density rho "Medium density";
- parameter Real kFixed(unit="")
+    annotation (Evaluate=true, Dialog(tab="Advanced"));
+  Medium.Density rho "Medium density";
+  final parameter Real kFixed(fixed=false)
```

Comment:
Why the attribute modifier `fixed=false`?  
Why don't you define the final value here instead in the `initial equation` section?

Reply:
Nice catch: this is a remaining from a previous implementation where an assert needed to be previously performed on `dpFixed_nominal` which was not an exposed parameter. I do not see a need here anymore.